packages/cli/src/commands/setup/deploy/providers/coherence.js

```diff
@@ -0,0 +1,112 @@
+// import terminalLink from 'terminal-link'
+import fs from 'fs'
+import path from 'path'
+
+import { getSchema, getConfig } from '@prisma/internals'
+import { Listr } from 'listr2'
+
+import { errorTelemetry } from '@redwoodjs/telemetry'
+
+import { getPaths, writeFilesTask } from '../../../../lib'
+import c from '../../../../lib/colors'
```

Comment:
We can start importing some of these functions from `@redwoodjs/cli-helpers` instead:

```suggestion
import { 
  colors as c,
  getPaths,
  writeFilesTask
} from '@redwoodjs/cli-helpers'
```